<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIModule</class>
 <widget class="QWidget" name="GUIModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Module</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Module Name: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameTxt">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Module file: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="fnameTxt">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Module Type: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="typeTxt">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="testedBox">
           <property name="text">
            <string>Tested</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="canCompileBox">
           <property name="text">
            <string>can compile?</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Weight: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="weightBar">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Completed: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="completedBar">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="functionsTable">
         <column>
          <property name="text">
           <string>Function Prototype</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Completed</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Weight</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tested</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="tasksTable">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <column>
          <property name="text">
           <string>Title</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Priority</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Kind</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>States</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Time Limit</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="notesTable">
         <column>
          <property name="text">
           <string>Note</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
